system_tests/bigquery.py

```diff
@@ -23,12 +23,15 @@
 
 
 _helpers.PROJECT = TESTS_PROJECT
-CLIENT = bigquery.Client()
```

Comment:
On a Travis build for a PR ([e.g.][1]) we used to run `run_system_tests.sh` and it would bail out since we only run system tests for merged commits to master (not PRs).

Now this check happens in Python.

This also means the Python script that does the environment checking imports `system_tests.run_system_test`. That import in turn imports `system_tests.datastore`, `system_tests.storage`, etc.

But a (security) feature of Travis means that PRs don't have the configuration needed to just call `datastore.Client()` and get all the implicit information.

[1]: https://travis-ci.org/GoogleCloudPlatform/gcloud-python/builds/100939831